<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1139</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_load">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>340</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>340</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Config</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_seq">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>340</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Noto Sans'; font-size:12pt; font-weight:600;&quot;&gt;Hardware Calibration&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ctrlHardware">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>420</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Control Hardware</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_SeqParams">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>821</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Sequence Parameters</string>
    </property>
    <widget class="QTabWidget" name="tabWidget_Seq">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>811</width>
       <height>321</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Imaging</string>
      </attribute>
      <widget class="QLabel" name="label_TR">
       <property name="geometry">
        <rect>
         <x>43</x>
         <y>220</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TR</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_loFreq">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>lo_freq (MHz)</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_rfDur">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>10</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_phDelay">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>90</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_rxTstart">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>50</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_Gamp">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>130</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_Shimx">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>20</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>RF duration</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_rfTstart">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>260</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>rf_tstart</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_phaseDur">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>140</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>phase duration</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_gradTstart">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>170</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_rfTend">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>20</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>rf_tend</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_trapRampDur">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>50</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_phAmp">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>210</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_phDur">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>130</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_rdAmp">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>250</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_phDelay">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>100</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>phase delay</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_rxTstart">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>60</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>rx_tstart</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_trapRampDur">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>60</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Trap ramp dur</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_rfTend">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>10</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_Gamp">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>G amp</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_rfTstart">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>250</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_rxPeriod">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>140</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>rx_period</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_trs">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>100</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>trs</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_loFreq">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_rfAmp">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_sliceAmp">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>180</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>slice amp</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_Gtstat">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>180</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>grad tstart</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_trs">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>90</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_NPointsy">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>220</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>phase amp</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_TR">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>210</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_rxPeriod">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>130</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_NPointsz">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>260</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>readout amp</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_sliceAmp">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>170</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_rxTend">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>90</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_rfAmp">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>54</width>
         <height>17</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>RF amp</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_rxTend">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>100</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>rx_tend</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_echosTR">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>170</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_echos_tr">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>180</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>echos per tr</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_echosDur">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>210</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_echosDur">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>220</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>echos duration</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_readOutDur">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>250</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_readoutDur">
       <property name="geometry">
        <rect>
         <x>378</x>
         <y>260</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>readout duration</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_phGint">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>10</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_phaseInter">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>20</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>phase grad interval</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_TrPauseDur">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>50</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_TRpauseDur">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>60</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>TR pause duration</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_readoutGradDur">
       <property name="geometry">
        <rect>
         <x>730</x>
         <y>90</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_readoutGradDur">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>100</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>readout grad duration</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget_seq">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_mplwindow1" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>381</width>
      <height>311</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout1"/>
   </widget>
   <widget class="QWidget" name="widget_mplwindow2" native="true">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>391</width>
      <height>311</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1139</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar_AcquisitionCtrl">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_connect"/>
   <addaction name="action_acquire"/>
   <addaction name="action_changeappearance"/>
   <addaction name="separator"/>
   <addaction name="action_gpacontroller"/>
   <addaction name="action_close"/>
  </widget>
  <action name="action_gpacontroller">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/waves.svg</normaloff>../resources/icons/waves.svg</iconset>
   </property>
   <property name="text">
    <string>GPA Controller</string>
   </property>
   <property name="toolTip">
    <string>Open GPA Waveform observer</string>
   </property>
  </action>
  <action name="action_savedata">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/clipboard.svg</normaloff>../resources/icons/clipboard.svg</iconset>
   </property>
   <property name="text">
    <string>Save Data</string>
   </property>
   <property name="toolTip">
    <string>Save Acquired Data</string>
   </property>
  </action>
  <action name="action_exportfigure">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/chart-line.svg</normaloff>../resources/icons/chart-line.svg</iconset>
   </property>
   <property name="text">
    <string>Export Figure</string>
   </property>
   <property name="toolTip">
    <string>Export the Figure of the Acquired Signal</string>
   </property>
  </action>
  <action name="action_focusfrequency">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/arrow-minimise.svg</normaloff>../resources/icons/arrow-minimise.svg</iconset>
   </property>
   <property name="text">
    <string>Focus Frequency</string>
   </property>
   <property name="toolTip">
    <string>Focus Actual Larmor Frequency</string>
   </property>
  </action>
  <action name="action_changeappearance">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/brush.svg</normaloff>../resources/icons/brush.svg</iconset>
   </property>
   <property name="text">
    <string>Appearance</string>
   </property>
   <property name="toolTip">
    <string>Change Window Appearance</string>
   </property>
  </action>
  <action name="action_connect">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/power-outline.svg</normaloff>../resources/icons/power-outline.svg</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect to marcos server</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="action_acquire">
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/social-youtube.svg</normaloff>../resources/icons/social-youtube.svg</iconset>
   </property>
   <property name="text">
    <string>Acquire</string>
   </property>
   <property name="toolTip">
    <string>Start Acquisition</string>
   </property>
  </action>
  <action name="action_close">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../resources/icons/delete-outline.svg</normaloff>../resources/icons/delete-outline.svg</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close PhisioMRI_GUI</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
